gc/2021/GC-07-13.md

```diff
@@ -0,0 +1,176 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the July 13 video call of WebAssembly's Garbage Collection Subgroup
+
+- **Where**: zoom.us
+- **When**: July 13, 4pm-5pm UTC (July 13, 9am-10am Pacific Daylight Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Fill out the form here to sign up if
+it's your first time: https://forms.gle/JehrAB4gWbtHjybt9. The meeting is open
+to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Discussion: Problems with nominal types [Andreas Rossberg] (1 hour)
+1. Closure
+
+## Meeting Notes
+
+### Introduction of attendees
+
+- Thomas Lively
+- Tim Steenvoorden
+- Manos Koukoutos
+- Ross Tate
+- Alon Zakai
+- Conrad Watt
+- Andreas Rossberg
+- Francis McCabe
+- Zalim Bashorov
+- Ryan Hunt
+- Emanuel Ziegler
+- Adam Klein
+- Daniel Wirtz
+- Asumu Takikawa
+- Slava Kuzmich
+- Jakob Kummerow
+- Keith Miller
+- Luke Wagner
+- Fil Pizlo
+- Deepti Gandluri
+- Petr Penzin
+- Paulo Matos
+- Ioanna Dimitriou
+
+
+### Discussion: Problems with nominal types [Andreas Rossberg] (1 hour)
+
+TODO: slides
```

Comment:
@rossberg, can you upload and link to your slides?

Reply:
Since they were already made, it would be useful to see the remainder of the slides ahead of next week's meeting. Knowing what's coming helps people know if a question or concern will be addressed in a later slide or whether it needs to be discussed at the moment while it is on topic, which makes for a more efficient meeting.